.bazeliskrc

```diff
@@ -0,0 +1 @@
+USE_BAZEL_VERSION=7.0.0
```

Comment:
We don't want to force the version. Generally locally we use the latest Bazel, but then the CI uses the oldest bazel version we support. Let's remove this and keep the github action specifying the version to use.

We also still support Bazel 6, so we want the CI to use Bazel 6.

Reply:
I have adapted the workflow to test v6 and v7